### PR TITLE
CEPH-83574331 - Adding new data pools to cephfs which has existing FS

### PR DESCRIPTION
**Pull request by hkadam134**
Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-8LPTTI/
Run Time : 4 mins, 18 secs
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/pacific/cephfs/tier-2_cephfs_test-volume-management.yaml [hunk 1]
```diff
@@ -273,4 +274,9 @@ tests:
       polarion-id: CEPH-83574165
       desc: cephfs subvolumegroup with different octal modes
       abort-on-fail: false
-
+  - test:
+      name: add datapools to existing FS
+      module: cephfs_vol_management.cephfs_vol_mgmt_add_datapool_to_existing_fs.py
+      polarion-id: CEPH-CEPH-83574331
```
Comment (veera-raghava-reddy) [inline suites/pacific/cephfs/tier-2_cephfs_test-volume-management.yaml:280]: CEPH-CEPH-8357433 - Check the Polarion ID?
CEPH is repeated twice.

Comment (veera-raghava-reddy) [reply suites/pacific/cephfs/tier-2_cephfs_test-volume-management.yaml:280]: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-CEPH-83574331
Work Item does not exist.

Comment (hkadam134) [reply suites/pacific/cephfs/tier-2_cephfs_test-volume-management.yaml:280]: It's there Veera : https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83574331

As CEPH-CEPH was repeated in the ID it did not show up for you..

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**